src/frontend/src/lib/components/dapps/DappsCarousel.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+	import { nonNullish } from '@dfinity/utils';
+	import Carousel from '$lib/components/carousel/Carousel.svelte';
+	import DappsCarouselSlide from '$lib/components/dapps/DappsCarouselSlide.svelte';
+	import {
+		dAppDescriptions,
+		type CarouselSlideOisyDappDescription
+	} from '$lib/types/oisyDappDescription';
+
+	export let styleClass: string | undefined = undefined;
+
+	const dappsCarouselSlides = dAppDescriptions.filter(({ carousel }) =>
```

Comment:
nitpick:  `dAppDescriptions.filter(nonNullish)`

does it work? eheh

Reply:
The filter only checks `carousel`, `.filter(nonNullish)` would look for the whole description to be non-nullish.

Reply:
ah right! all good! tks!